### DISCUSSION TITLE
yespowerR16 not working

### DISCUSSION THREAD

**Discussion by lurchinms** [coinfoundry/miningcore#1713] [General]
Pool will not start with 
"yenten": {
        "name": "Yenten",
        "canonicalName": "Yenten",
        "symbol": "YTN",
        "family": "bitcoin",
        "website": "http://yentencoin.info/",
        "market": "https://coinmarketcap.com/currencies/yenten",
        "twitter": "https://twitter.com/yentencoin",
        "telegram": "https://t.me/yenten",
        "discord": "https://discord.gg/RTbPxu3",
        "coinbaseHasher": {
            "hash": "sha256d"
        },
        "headerHasher": {
            "hash": "yespowerR16"
        },
        "blockHasher": {
            "hash": "reverse",
            "args": [ { "hash": "sha256d" } ]
        },
        "shareMultiplier": 1048576,
        "explorerBlockLink": "http://ytn.ccore.online/block/$height$",
        "explorerTxLink": "http://ytn.ccore.online/transaction/{0}",
        "explorerAccountLink": "http://ytn.ccore.online/address/{0}"

**Comment by xiaolin1579** [comment]
Try
```
    "yenten": {
        "name": "Yenten",
        "canonicalName": "Yenten",
        "symbol": "YTN",
        "family": "bitcoin",
        "website": "http://yentencoin.info/",
        "twitter": "https://twitter.com/yentencoin",
        "telegram": "https://t.me/yenten",
        "discord": "https://discord.gg/RTbPxu3",
        "coinbaseHasher": {
            "hash": "sha256d"
        },
        "headerHasher": {
            "hash": "yespowerr16"
        },
        "blockHasher": {
            "hash": "reverse",
            "args": [
                {
                    "hash": "sha256d"
                }
            ]
        },
        "shareMultiplier": 65536,
        "explorerBlockLink": "http://ytn.ccore.online/block/$height$",
        "explorerTxLink": "http://ytn.ccore.online/transaction/{0}",
        "explorerAccountLink": "http://ytn.ccore.online/address/{0}"
    },

```
